index.html

```diff
@@ -6,16 +6,35 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Alex's Portifolio</title>
     <link rel="stylesheet" href="styles.css" />
-    <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons"
+    <link
+      rel="stylesheet"
+      href="https://fonts.googleapis.com/icon?family=Material+Icons"
     />
   </head>
   <body class="container bod">
     <header class="container navigation">
-      <p class="logo"><a href="index.html">John Doe</a></p>
-      <p class="logo"><a href="index.html">Portfolio</a></p>
-      <p class="logo"><a href="index.html">About</a></p>
-      <p class="logo"><a href="index.html">Contact</a></p>
-      <i class="material-icons">menu</i>
+      <p class="logo"><a class="menuitem1" href="index.html">John Doe</a></p>
+      <p class="logo"><a href="#portfolio">Portfolio</a></p>
+      <p class="logo"><a href="#skills">About</a></p>
+      <p class="logo"><a href="#contact">Contact</a></p>
+      <i class="material-icons menuIcon">menu</i>
+      <ul class="hidden">
+        <li>
+          <i class="material-icons closeIcon">close</i>
+        </li>
+        <li class="container menuItem">
+          <a href="#portfolio">Portfolio</a>
+          <i class="material-icons">chevron_right</i>
+        </li>
+        <li class="container menuItem">
+          <a href="#skills">About </a>
+          <i class="material-icons">chevron_right</i>
+        </li>
+        <li class="container menuItem">
+          <a href="#contact">Contact </a>
+          <i class="material-icons">chevron_right</i>
+        </li>
+      </ul>
```

Comment:
- [ ] When the user click (or tap) any option of the mobile menu, the mobile menu disappear. Right now, the menu stays there, please fix it so that if user wants to go to any section they will go right away in one click and not one more close button click.